qiskit/transpiler/passes/synthesis/hls_plugins.py

```diff
@@ -214,6 +214,41 @@
    MCXSynthesisNDirtyI15
    MCXSynthesis1CleanB95
    MCXSynthesisDefault
+
+MCMT Synthesis
+''''''''''''''
+
+.. list-table:: Plugins for :class:`.MCMTGate` (key = ``"mcmt"``)
+    :header-rows: 1
+
+    * - Plugin name
+      - Plugin class
+      - Number of clean ancillas
+      - Number of dirty ancillas
+      - Description
+    * - ``"vchain"``
+      - :class:`.MCMTSynthesisVChain`
+      - `k-1`
+      - `0`
+      - uses a linear number of Toffoli gates
+    * - ``"noaux_v24"``
```

Comment:
I think it should be only noaux (without the v24)

Reply:
Good catch!
```suggestion
    * - ``"noaux"``
```